app/views/components/answer_section.scala.html

```diff
@@ -19,6 +19,10 @@
 
 @(answerSection: AnswerSection)(implicit messages: Messages)
 
+@if(answerSection.sectionKey.isDefined){
+    <h2 class="heading-large">@messages(answerSection.sectionKey.get)</h2>
```

Comment:
Is it safe to call `.get`. I don't think so?

Reply:
the previous line checks that the sectionKey cannot be None ?

Reply:
That's fine 👍